Uebersicht/UBPreferencesController.xib

```diff
@@ -133,6 +142,26 @@
                             <color key="backgroundColor" name="controlColor" catalog="System" colorSpace="catalog"/>
                         </textFieldCell>
                     </textField>
+                    <button fixedFrame="YES" translatesAutoresizingMaskIntoConstraints="NO" id="4Cm-6a-MW9">
+                        <rect key="frame" x="220" y="265" width="182" height="18"/>
+                        <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMinY="YES"/>
+                        <buttonCell key="cell" type="check" title="Enable security measures" bezelStyle="regularSquare" imagePosition="left" state="on" inset="2" id="6iu-kH-BSa">
```

Comment:
Would you prefer to have one setting for each security measure?

Also, if it's fine to have one checkbox, is there a better wording?

Reply:
I need to think about it a bit as well. Worst case I can play with it before releasing, so let's just leave as is for now.